scripts/ci/build_emscripten.sh

```diff
@@ -62,11 +62,12 @@ then
 	echo -n "$CIRCLE_SHA1" >commit_hash.txt
 fi
 
-# Disable warnings for unqualified "move()" calls, introduced and enabled by
+# Disable warnings for unqualified `move()` calls, introduced and enabled by
 # default in clang-16 which is what the emscripten docker image uses.
 # Additionally, disable the warning for unknown warnings here, as this script is
 # also used with earlier clang versions.
-CMAKE_CXX_FLAGS="-Wno-unqualified-std-cast-call -Wno-unknown-warning-option"
+# TODO: This can be removed if and when all usages of `move()` use the `std::` qualifier.
+CMAKE_CXX_FLAGS="-Wno-unqualified-std-cast-call"
```

Comment:
But this is not part of the image?

Reply:
To expand on that: This is called by the CI in the environment of the docker image, but it's not part of it.

Reply:
This can also be seen here: https://app.circleci.com/pipelines/github/ethereum/solidity/26062/workflows/a9734583-6920-4572-a62d-35efae37d0f7/jobs/1151467 (step "build", line 52)

Reply:
Ah, good. One less problem :)